tests/models/registry.py

```diff
@@ -295,6 +295,7 @@ def check_available_online(
         "internlm/internlm3-8b-instruct", trust_remote_code=True
     ),
     "JAISLMHeadModel": _HfExamplesInfo("inceptionai/jais-13b-chat"),
+    "Jais2CausalLM": _HfExamplesInfo("inceptionai/Jais-2-8B-Chat"),
```

Comment:
```suggestion
    "Jais2ForCausalLM": _HfExamplesInfo("inceptionai/Jais-2-8B-Chat"),
```

You made a typo here

Reply:
And also you should add `min_transformers_version` since it's not available in v4.57

Reply:
the typo was corrected by sarath about 3 hours ago. Should we rebase ?

Reply:
oh sorry you are talking about the min_transformers_version !

Reply:
thanks @alielfilali42 for chiming in.. @DarkLight1337 i have updated the min_transformers_version="4.58" looking a current available release being 4.57.3. Hope this is correct.